bindings/rust/bench/README.md

```diff
@@ -0,0 +1,43 @@
+# Benchmarking s2n-tls
+
+We benchmark s2n-tls against two other common TLS libraries, Rustls and OpenSSL. We use Rust bindings for s2n-tls and OpenSSL and Criterion to benchmark these libraries. 
+
+## Setup 
+
+Setup is easy! Just run `/bindings/rust/generate.sh` to generate Rust bindings for s2n-tls and have OpenSSL installed.
```

Comment:
Is the OpenSSL install necessary for generating the bindings or for running the tests? Here it's hard to tell.

Reply:
Updated wording